tests/core/unstructured_vespa_index/test_schemas/unstructured_vespa_index_schema.sd

```diff
@@ -118,8 +118,8 @@ schema marqo__test_00unstructured_00schema {
             query(marqo__add_weights) tensor<double>(p{})
         }
         function modify(score) {
-            expression: if (count(query(marqo__mult_weights)) == 0, 1, reduce(query(marqo__mult_weights) * attribute(marqo__score_modifiers), prod)) * score + reduce(query(marqo__add_weights) * attribute(marqo__score_modifiers), sum)
-       }
+            expression: if (count(query(marqo__mult_weights)) == 0, 1, if (count(query(marqo__mult_weights) * attribute(marqo__score_modifiers)) == 0, 1, reduce(query(marqo__mult_weights) * attribute(marqo__score_modifiers), prod))) * score + if (count(query(marqo__add_weights)) == 0, 0, reduce(query(marqo__add_weights) * attribute(marqo__score_modifiers), sum))
```

Comment:
Why not use something identical to structured?

Reply:
Because in unstructured we only have one tensor (which is double). The expression in structured uses two tensors